pkg/bucket/replication/replication.go

```diff
@@ -176,6 +188,9 @@ func (c Config) Replicate(obj ObjectOpts) bool {
 		if rule.Status == Disabled {
 			continue
 		}
+		if obj.Replica && rule.SourceSelectionCriteria.ReplicaModifications.Status == Enabled {
+			return true
+		}
 		return true
```

Comment:
why does this `if` condition needed if we are returning like 

```
if this {
   return true
}
return true
```

```
return true
```

is the same